nodebuilder/header/config.go

```diff
@@ -43,13 +43,19 @@ func DefaultConfig(tp node.Type) Config {
 		Store:        store.DefaultParameters(),
 		Syncer:       sync.DefaultParameters(),
 		Server:       p2p_exchange.DefaultServerParameters(),
+		Client:       p2p_exchange.DefaultClientParameters(),
 	}
 
 	switch tp {
 	case node.Bridge:
+		cfg.Client = p2p_exchange.ClientParameters{}
```

Comment:
why is this necessary?

Reply:
I just dumbly followed the behavior that was before, where Bridge had empty Client params. But now I think it is fine to remove it.

Reply:
Ah wait. It may actually impact the `celestia init`. Please check on the BN if the client config is initialized there.

Reply:
@renaynay

Reply:
```
	// we do not create a client for bridge nodes
	if tp == node.Bridge {
		return nil
	}
```

so no